core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/projection/ProjectionOpDesc.scala

```diff
@@ -45,4 +51,14 @@ class ProjectionOpDesc extends MapOpDesc {
       )
       .build()
   }
+
+  override def getOperatorVersion(): String = {
+    val operatorVersionPath = Paths.get("operator_version.properties").toAbsolutePath()
+    val props = new Properties
+    val fileStream = Files.newInputStream(operatorVersionPath)
+    props.load(fileStream)
+    fileStream.close()
+    val operatorVersionMap = props.asScala.toMap
+    operatorVersionMap("Projection")
```

Comment:
Instead of hardcoding this way lets use .getclass().getsimplifiedname() something similar

Reply:
fixed